lib/packwerk/package_set.rb

```diff
@@ -27,9 +27,11 @@ def load_all_from(root_path, package_pathspec: nil)
 
       def package_paths(root_path, package_pathspec)
         bundle_path_match = Bundler.bundle_path.join("**").to_s
-
-        Dir.glob(File.join(root_path, package_pathspec, PACKAGE_CONFIG_FILENAME))
```

Comment:
This doesn't work when more than one package path is provided, as an array.

Reply:
Do we support an array of paths or are you adding a feature here?

Reply:
The plural form of the configuration key, `package_paths`, tells me that more than one path could be provided.  It also feels like something we should support, especially once we fix the existing logic.

But `File.join(root_path, package_pathspec, PACKAGE_CONFIG_FILENAME)` will not produce a viable path if `package_pathspec` is an array nor a comma-delimited set of paths.